openshift/3scale_backend.conf

```diff
@@ -34,6 +34,8 @@ ThreeScale::Backend.configure do |config|
   config.kinesis_stream_name = "#{ENV['CONFIG_KINESIS_STREAM_NAME']}"
   config.kinesis_region = "#{ENV['CONFIG_KINESIS_REGION']}"
   config.stats.bucket_size = ENV['CONFIG_STATS_BUCKET_SIZE'].to_i
+  config.stats.delete_batch_size = "#{ENV['CONFIG_DELETE_STATS_BATCH_SIZE']}"
```

Comment:
Should we use `parse_int_env` here? Like in other int envs above like `CONFIG_REDIS_CONNECT_TIMEOUT`, `CONFIG_REDIS_WRITE_TIMEOUT`, etc.

Reply:
it is parsed here, in `Configuration` class and assigned default value if empty: https://github.com/3scale/apisonator/pull/73/files#diff-5c109eb6e5b153d75018f097b9887048R111

Reply:
:+1: